include/distr_queue.h

```diff
@@ -116,16 +112,36 @@ class distr_queue {
 	}
 
   private:
-	std::shared_ptr<detail::distr_queue_tracker> m_tracker;
-
-	void init(detail::device_or_selector device_or_selector) {
-		if(!detail::runtime::is_initialized()) { detail::runtime::init(nullptr, nullptr, device_or_selector); }
-		try {
-			detail::runtime::get_instance().startup();
-		} catch(detail::runtime_already_started_error&) {
-			throw std::runtime_error("Only one celerity::distr_queue can be created per process (but it can be copied!)");
+	struct tracker {
```

Comment:
I think we'd want a brief comment here on what the purpose of the `distr_queue::tracker` struct is. It's not user-facing, but for internal documentation.